ci/builders/local_engine.json

```
@@ -184,16 +184,17 @@
         "--simulator-cpu=arm64",
         "--runtime-mode",
         "debug",
+        "--unoptimized",
         "--no-stripped",
         "--no-lto",
         "--xcode-symlinks",
         "--rbe",
         "--no-goma"
       ],
-      "name": "macos/ios_debug_sim_arm64",
+      "name": "macos/ios_debug_sim_unopt_arm64",
       "description": "Builds a debug mode engine that targets the arm64 iOS simulator from a macOS host.",
```

To match the other builds:
```suggestion
      "description": "Builds an unoptimized debug mode engine that targets the arm64 iOS simulator from a macOS host.",
```

---

Nice catch: https://github.com/flutter/engine/pull/55498